### ISSUE TITLE
task: add two way sync for model attributes updated when interacting with canvas

### ISSUE THREAD

**Issue by alisterburt** [tlambert03/microvis#22]
Currently, zooming in the vispy canvas does not change `Camera.zoom`, we should make sure two way communication is working here

**Comment by tlambert03** [comment]
Yeah that's a big category!  Also worth noting that any programmatic changes made to the native object won't be reflected in the model
